### PR TITLE
Create RawHTML component

### PR DESCRIPTION
**Pull request by BessieSteinberg**

### CODE REVIEW

File: src/components/JournalPage/index.jsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import {
   IMAGE,
 } from '../../data/constants/contentTypes';
 
+import RawHTML from '../../components/RawHTML';
```
Comment (tuchfarber) [inline src/components/JournalPage/index.jsx]: You're already in components, so you really just need a `../RawHTML`

File: src/components/JournalPage/index.jsx [hunk 2]
```diff
@@ -29,7 +31,7 @@ class JournalPage extends React.Component {
                   return <div>RichText</div>;
                 case RAW_HTML:
                   // return <RawHTML />
-                  return <div>RawHTML</div>;
+                  return <RawHTML content={el.value} />;
```
Comment (bill-filler) [inline src/components/JournalPage/index.jsx]: maybe change the case statement so RICH_TEXT and RAW_HTML both go to the RawHTML component as they should both render the same. Can do that in another PR if preferred
